### PR TITLE
Test ebad

### PR DESCRIPTION
**Pull request by ebadkamil**
1) metadata dictionary should look like?

{''*/DET/0CH*": {'source': xxx , 'timestamp.tid':xxx, 'timestamp.frac': xxx, 'timestamp.sec':xxx,   }}

2)
metadata[src]['timestamp.tid'] = int(metadata[src]['timestamp.tid'])

if this is not implemented then C++ client says "timestamp.tid" is of MAP type.

### CODE REVIEW

**Comment by zhujun98** [comment]
```
metadata[src]['timestamp.tid'] = int(metadata[src]['timestamp.tid'])

if this is not implemented then C++ client says "timestamp.tid" is of MAP type.
```
I don't understand what do you mean byb MAP type

metadata
--------
source, , , string
timestamp, , , double
timestamp.frac, , , string
timestamp.sec, , , string
timestamp.tid, , , uint64_t

**Comment by ebadkamil** [comment]
> ```
> metadata[src]['timestamp.tid'] = int(metadata[src]['timestamp.tid'])
> 
> if this is not implemented then C++ client says "timestamp.tid" is of MAP type.
> ```
> 
> I don't understand what do you mean by MAP type
> ## metadata
> 
> source, , , string
> timestamp, , , double
> timestamp.frac, , , string
> timestamp.sec, , , string
> timestamp.tid, , , uint64_t


accessing the containerType via 
for(auto &v : data.metadata)
{
auto container = v.second.containerType(); // returns "MAP" when key ( v.first) is  'timestamp.tid'
}

**Comment by zhujun98** [comment]
```
accessing the containerType via 
for(auto &v : data.metadata)
{
auto container = v.second.containerType(); // returns "MAP" when key ( v.first) is  'timestamp.tid'
}
```
It returns an empty string because it is a scalar data. I just double checked it.

File: karabo_data/export.py [hunk 1]
```diff
@@ -117,20 +117,32 @@ def stop(self):
             self._interface = None
 
     def _serialize(self, data, metadata=None):
+
+        for key,val in data.items():
+            if 'metadata' in val:
+                val['metadata']['timestamp.tid'] = int(val['metadata']['timestamp.tid'])
```
Comment (takluyver) [inline karabo_data/export.py]: This chunk can be dropped now - I fixed it in a different place in #79.

**Pull request merged**